### ISSUE TITLE
Sniff for getimagesize()

### ISSUE THREAD

**Issue by davidsword** [Automattic/VIP-Coding-Standards#685]
## What problem would the enhancement address for VIP?

`getimagesize()` holds up the PHP process downloading the entire image to just read its file size.

## Describe the solution you'd like

Though a wrapper function could be used instead [to download only first few bytes of image](https://www.php.net/manual/en/function.getimagesize.php#88793), most images in WordPress have width and height attributes already, like from wp_get_attachment_metadata() & that existing data should be used instead. 

If the image is external and not in WordPress' media library, `getimagesize()` should not be used and any sizing could be done with JS/CSS instead, or `getimagesize()` could used with extreme caution warning images need to be downloaded fully to get the image size.

## What code should be reported as a violation?

```php
list($wdth, $hght) = getimagesize( $image_url );
```
